Makefile

```diff
@@ -0,0 +1,11 @@
+run:
```

Comment:
**suggestion**: I don't use this pattern too much (usually I type the entire command), but it may be useful for other developers. I will just suggest that you include a command to run the tests too.